package.json

```diff
@@ -8,7 +8,8 @@
     "clean": "rm -rf build node_modules www platforms plugins",
     "lint": "npm run lint:ts",
     "lint:ts": "tslint 'src/**/*.ts'",
-    "reset": "npm run clean && npm ci"
+    "reset": "npm run clean && npm ci",
+    "storybook": "web-dev-server"
```

Comment:
how would you feel about this being like... `npm run action src/www/start storybook`?

Reply:
Don't we need to specify what path to open somehow?

If we go with an action, we should probably place it in the ui_components/ folder

Reply:
yeah this will change when i move it, thanks